### PR TITLE
[WIP] Test for Python 3.7 as well

### PR DESCRIPTION
**Pull request by zuphilip**
Travis is currently failing for Python 3.7 https://travis-ci.org/UB-Mannheim/hocr-tools/builds/425885371

### CODE REVIEW

**Comment by zuphilip** [comment]
Oh, maybe Travis is no yet ready for Python 3.7 or it is more advance to add it? CC @kba

**Comment by stweil** [comment]
Python 3.7 is now found, but Travis fails later: https://travis-ci.org/UB-Mannheim/hocr-tools/jobs/426010558.

**Comment by zuphilip** [comment]
Hm.. so building `lxml` fails. I don't know how to fix that... :worried:

**Comment by stweil** [comment]
I modified the CI to use the Python packages from the distribution, and that obviously works. Now we have to fix my hack to restrict it to Python 3.7 and re-enable the older versions.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
 cache:
   directories:
     - /home/travis/virtualenv/python2.7/lib/python2.7/site-packages
     - /home/travis/virtualenv/python3.4/lib/python3.4/site-packages
     - /home/travis/virtualenv/python3.5/lib/python3.5/site-packages
     - /home/travis/virtualenv/python3.6/lib/python3.6/site-packages
+    - /home/travis/virtualenv/python3.7/lib/python3.7/site-packages
 install:
-  - pip install -r requirements.txt
+  - sudo apt install python3-lxml python3-pil python3-reportlab
```
Comment (zuphilip) [inline .travis.yml]: It is suggested to not use `apt` but `pip` for installing python packages: https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs

**Comment by stweil** [comment]
This was superseded by #139, so I am closing it.

**Pull request closed without merging**